polaris-tokens/src/utilities.ts

```diff
@@ -210,11 +210,12 @@ function getUpMediaCondition(breakpoint: string) {
 }
 
 /**
- * Down media condition breakpoints are being subtracted by 0.05px to prevent
+ * Down media condition breakpoints are being subtracted by 0.04px to prevent
  * them from overwriting up media queries. We experimented with multiple offsets
- * and felt that 0.05px would be the safest across different pixel densities.
+ * and felt that 0.04px would be the safest across different pixel densities,
+ * while being representable in ems with 4 decimal places of precision.
  */
 function getDownMediaCondition(breakpoint: string) {
-  const offsetBreakpoint = parseFloat(toPx(breakpoint) ?? '') - 0.05;
+  const offsetBreakpoint = parseFloat(toPx(breakpoint) ?? '') - 0.04;
```

Comment:
Thoughts on leaving the `0.05px` offset and adjusting the precision? e.g.

<img width="575" alt="image" src="https://user-images.githubusercontent.com/32409546/192040860-15028fae-6740-476d-8a52-5723833c0ac5.png">

Reply:
Such good points! Totally agree 👍